pkg/aws/naming.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2023 The Paraglider Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+)
+
+const (
+	paragliderPrefix         = "para"    // Prefix for all resources
+	defaultSecurityGroupName = "default" // Default security group name as set by AWS
+)
+
+// getDescribeFilter returns a filter for a resource to use for getting (e.g., Describe...).
+func getDescribeFilter(namespace string, name string) []types.Filter {
+	return []types.Filter{
+		{Name: aws.String("tag:Name"), Values: []string{name}},
+		{Name: aws.String("tag:Namespace"), Values: []string{namespace}},
+	}
+}
+
+// getCreateTagSpecifications returns tag specifications for a resource to use for creating (e.g., Create...).
+func getCreateTagSpecifications(namespace string, name string, resourceType types.ResourceType) []types.TagSpecification {
```

Comment:
This naming is just a bit awkward to me. Maybe something like `getIntitialParagliderResourceTags` or something like that?